package.json

```diff
@@ -6,5 +6,8 @@
   },
   "scripts": {
     "prepare": "husky install"
+  },
+  "dependencies": {
+    "react-dropzone": "^11.4.2"
```

Comment:
This is installed in the root directory and not the client folder, please revert.

Reply:
And add it to the client package so other devs can get the dependency when they run `yarn install`

Reply:
I had fixed a lot of the comments in the other issue during final integration, can I delete this issue and get the other one approved?

Reply:
That's fine